### PR TITLE
Add Github workflows based on munigeo

### PR DESCRIPTION
**Pull request by voneiden**
The priority is to get the release pipeline going. Black, flake8 and isort will be enabled in a separate dedicated task.

### CODE REVIEW

**Comment by voneiden** [comment]
Converting to draft until I manage to fix the tests.

File: .github/workflows/ci.yml
Comment (danipran) [inline .github/workflows/ci.yml:77]: Mismatching versions (3.9 above, 3.8 here). I know it's commented out, but if I went with the TODO comment and just enabled this later, it might go unnoticed.

File: .github/workflows/ci.yml
Comment (danipran) [inline .github/workflows/ci.yml:20]: The package doesn't seem to "officially" support 3.9 yet (according to `setup.py` and `.travis.yml`). I'm all for 3.9 support, but probably should stick to 3.8 for now.

Also applies to other similar lines in `ci.yml`, `release.yml` and `staging.yml`.

**Review by danipran — CHANGES_REQUESTED** [review]
Python version shenanigans!

File: .github/workflows/ci.yml
Comment (danipran) [reply .github/workflows/ci.yml:77]: Well, in hindsight, this is probably not relevant, seeing that I'm suggesting to change the other lines to 3.8 instead...

File: .github/workflows/ci.yml
Comment (voneiden) [reply .github/workflows/ci.yml:20]: Tangentially related: .travis.yml can probably be removed? https://app.travis-ci.com/github/City-of-Helsinki/django-orghierarchy

Comment (danipran) [reply .github/workflows/ci.yml:20]: ^ @tuomas777 Any idea if Travis is relevant anymore?

File: .github/workflows/ci.yml
Comment (charn) [inline .github/workflows/ci.yml:101]: v2

File: .github/workflows/ci.yml
Comment (charn) [inline .github/workflows/ci.yml:47]: v4

File: .github/workflows/ci.yml
Comment (charn) [inline .github/workflows/ci.yml:45]: v3

File: .github/workflows/ci.yml
Comment (charn) [reply .github/workflows/ci.yml:47]: Well, I guess you can quickly check the versions used in other GH actions as well.

Comment (voneiden) [reply .github/workflows/ci.yml:47]: I'll also make a task for updating munigeo

**Review by danipran — APPROVED** [review]
LGTM :+1: should probably get Juha's approval as well before merging 🤔

**Comment by voneiden** [comment]
One approver to blame for all.

@charn can file a formal complaint retroactively if needs be. I take the "leave comment" review generally as "proceed once issues are addressed".

**Pull request merged**
